### ISSUE TITLE
Option to set the MLM_LICENSE_FILE when starting the Matlab kernel in Jupyter.

### ISSUE THREAD

**Issue by lekshmideepu** [mathworks/jupyter-matlab-proxy#3]
When starting the Matlab kernel in Jupyter, it asks to enter the license information. As of now, one could only enter the information as "port@hostname" which was not working unfortunately in our case for the network license manager. It would also work by setting the MLM_LICENSE_FILE. As a workaround we had added the path to the license file in the below line and it worked like charm. 

https://github.com/mathworks/jupyter-matlab-proxy/blob/main/jupyter_matlab_proxy/app_state.py#L386

So I would like to request you to have another option to enter the path to the license file in the jupyter-matlab interface.

**Comment by prabhakk-mw** [comment]
Hi, @lekshmideepu 

Thank you for reporting the issue, and suggesting the enhancement.
While we work on incorporating the enhancement, could you please share more information as to why the network license manager was failing to license MATLAB at your end? Any logs or error messages would help us investigate.

**Comment by lekshmideepu** [comment]
Thanks for the response. I am not very sure why it was failing. started Jupyterhub on our cluster and the network license manager is configured on a server triad. I checked the license manager and it was up and running. I was able to ping this machine from the cluster.  When I try to start the "MATLAB" session in the Jupiter interface, it says "MATLAB Status: Not running".  "Error: Communication with server failed"
